app/controllers/api/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2012 Brian Ploetz (bploetz@gmail.com)
```

Comment:
I'll try to make it a bit clearer this time: There is no need to include licenses from Gems we use. License information is bundled within the Gem itself so we do not have to take care if that. If you added that license because you included Versionist in our Gemfile, then there is no need.

The comment mentioned by Jonne was me getting angry at @AugierLe42e because he copied files from [nov/openid_connect_sample:app/controllers/connect](https://github.com/nov/openid_connect_sample/tree/master/app/controllers/connect) into our codebase. I have done a quick validation and as far as I can tell, all those files are now removed or superseded by own work, so, if this is true, there is also no reason to include the license of nov's repo. One should do a more careful checking though.